File: IntroPC/Tarea 5/Fgrav/fuerza-grav.cpp
#include <cstdio>
#include <cmath>
using namespace std;

int main(void)
{
  float m1,m2,r,G,grav;
  printf("ingrese la masa 1: ");
  scanf("%e",&m1);
  printf("\n");
  printf("ingrese la masa 2: ");
  scanf("%f",&m2);
  printf("\n");
  printf("por ultimo ingresela distancia entre ambos cuerpos: ");
  scanf("%f",&r);
  printf("\n");
  G=6.674*pow(10.0,-11);
  grav=(G*((m1*m2)/(pow(r,2))));
  printf("la fuerza de atraccion entre ambos cuerspos se de: %e \n ",grav); 
  return 0;

}

File: IntroPC/Tarea 5/Fgrav/fuerza-grav.py
#!/usr/bin/env python



m1=input("ingrese la masa 1: ")
m2=input("ingrese la masa 2: ")
r=input("por ultimo ingresela distancia entre ambos cuerpos: ")
G=6.674E-11
grav=(G*((m1*m2)/r**2))
print"la fuerza de atraccion entre ambos cuerspos se de: %f",(grav)
